lib/globals/_theme.scss

```diff
@@ -55,3 +56,14 @@ $dark-hot-pink-20: rgba(225, 0, 80, 0.2);
 $orange-yellow-20: rgba(255, 165, 0, 0.2);
 $battleship-grey-20: rgba(108, 117, 125, 0.2);
 $dark-hot-pink-50: rgba(225, 0, 80, 0.5);
+
+$shimmer-color-light: #f6f7f8;
+$shimmer-color-dark: #edeef1;
+
+// *********** BOX SHADOWS ****************
```

Comment:
Comments for single lines can be provided at the end of the line. Comments like this increase the code length (Also try avoiding unnecessary line gaps). Moreover, The names can be descriptive to be understandable at first glance, so probably we can remove comments altogether and just use names that are descriptive enough.

Reply:
321bd4ced83925829ae7a889f3d16964b6fa0c12